iphone/Maps/Classes/CustomViews/NavigationDashboard/Views/MWMNavigationInfoView.mm

```diff
@@ -15,16 +15,16 @@
 #include "geometry/angles.hpp"
 
 namespace {
-CGFloat const kTurnsiPhoneWidth = 96;
-CGFloat const kTurnsiPadWidth = 140;
-
-CGFloat const kSearchButtonsViewHeightPortrait = 200;
-CGFloat const kSearchButtonsViewWidthPortrait = 200;
-CGFloat const kSearchButtonsViewHeightLandscape = 56;
-CGFloat const kSearchButtonsViewWidthLandscape = 286;
-CGFloat const kSearchButtonsSideSize = 44;
-CGFloat const kBaseTurnsTopOffset = 28;
-CGFloat const kShiftedTurnsTopOffset = 8;
+constexpr CGFloat kTurnsiPhoneWidth = 96;
+constexpr CGFloat kTurnsiPadWidth = 140;
```

Comment:
nit: If you have changed here, I'd put { from the new line and put constexpr on the right as below.

Reply:
ObjC braces start on the same line, but ok )